cmd/crates/soroban-test/tests/it/integration/util.rs

```diff
@@ -33,6 +34,10 @@ pub async fn deploy_swap(sandbox: &TestEnv) -> String {
     deploy_contract(sandbox, SWAP).await
 }
 
+pub async fn deploy_custom_account(sandbox: &TestEnv) -> String {
```

Comment:
Not for this PR, but a good small one you could do is add `deploy` to `Wasm` that takes a `&TestEnv`.
This we don't need a function per file.  e.g. `HELLO_WORLD.deploy(sandbox)`.

Reply:
Okay, noted